test/torch_np/numpy_tests/core/test_indexing.py

```diff
@@ -602,16 +602,21 @@ def test_boolean_index_cast_assign(self):
         zero_array[bool_index] = np.array([1])
         assert_equal(zero_array[0, 1], 1)
 
+        # np.ComplexWarning moved to np.exceptions in numpy>=2.0.0
+        # np.exceptions only available in numpy>=1.25.0
+        has_exceptions_mod = getattr(np, "exceptions", None)
```

Comment:
nit: use hasattr?

Reply:
thanks for the suggestion, updated, the PR.